### ISSUE TITLE
Unable to install Testkube in MacOS Monterey (12.0.1) / Minikube v1.24.0

### ISSUE THREAD

**Issue by chlete** [kubeshop/testkube#483]
**Describe the bug**
After downloading the plugin, I'm unable to install testkube. 

**To Reproduce**
1. Run `bash < <(curl -sSLf https://kubeshop.github.io/testkube/install.sh )`
2. Run `kubectl testkube install`
3. See error:
```
getting server info: services "testkube-api-server" not found

████████ ███████ ███████ ████████ ██   ██ ██    ██ ██████  ███████
   ██    ██      ██         ██    ██  ██  ██    ██ ██   ██ ██
   ██    █████   ███████    ██    █████   ██    ██ ██████  █████
   ██    ██           ██    ██    ██  ██  ██    ██ ██   ██ ██
   ██    ███████ ███████    ██    ██   ██  ██████  ██████  ███████
                                           /tɛst kjub/ by Kubeshop


⨯ adding testkube repo (error: process error: exec: not started
output: )

⨯ updating helm repositories (error: process error: exec: not started
output: )

```
The problem seems to be that the processes is not started (`process error: exec:`). I suspect MacOS might not start it due to some security contraint. Unfortunately there is not a lot of information after the `output:` clause.

**Expected behaviour**
Testkube is installed.

**Version / Cluster**
- Testkube: 0.6.4
- K8s: Minikube
- K8s version: 1.22.3

**Screenshots**
If applicable, add CLI commands/output to help explain your problem.
 
**Additional context**

**Comment by chlete** [comment]
I found the problem, `helm` was not installed. I suggest to improve the detection of the presence of helm, and output when not found :)

**Comment by exu** [comment]
@chlete thanks for finding this one, we'll fix this in 0.7 release.
